query-engine/connector-test-kit-rs/query-engine-tests/src/utils/querying.rs

```diff
@@ -115,3 +115,16 @@ macro_rules! retry {
         }
     }};
 }
+
+#[macro_export]
+macro_rules! with_id_excess {
+    ($runner:expr, $query_template:expr) => {{
+        let max_bind_values = $runner
+            .max_bind_values()
+            .expect("Test expected to run only for relational databases.");
+
+        let cycle = |argn: usize| (argn + 1 % 10).to_string();
```

Comment:
This won't cycle, but will always return the argument incremented by 1. Should be this instead?
```suggestion
        let cycle = |argn: usize| ((argn + 1) % 10).to_string();
```

Reply:
I mean, should the priority of the operations be fixed here, or is the cycling behaviour not necessary anymore in the latest version of changes and this should be just `argn + 1`?

Reply:
Wait, what? no, both are wrong. It should be (argn % 10) + 1 (Wanting ids 1..10)